src/server/redis_connection.h

```diff
@@ -65,6 +65,9 @@ class Connection : public EvbufCallbackBase<Connection> {
   RESP GetProtocolVersion() const { return protocol_version_; }
   void SetProtocolVersion(RESP version) { protocol_version_ = version; }
   std::string Bool(bool b) const;
+  std::string BigNumber(const std::string &n) const {
+    return protocol_version_ == RESP::v3 ? "(" + n + CRLF : BulkString(n);
```

Comment:
Could SimpleString be better here?

Reply:
Redis is using the bulk string in RESP v2, so it'd be better to align with it. And the simple string is generally used when representing the status.